AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -210,6 +210,10 @@ extension BluetoothManager: CBCentralManagerDelegate {
 
         guard mobilePeripheralSessionManager.activeSessionInProgressWith(peripheral) else { return }
         mobilePeripheralSessionManager.markActiveSessionAsDisconnected(peripheral: peripheral)
+        connectWithTimeout(using: peripheral)
```

Comment:
Please remove those changes so that there are no conflicts with BT manager (this class is getting deleted anyway)